# The origins and genetic interactions of KRAS mutations are allele- and tissue-specific

summarization method: T5

## Introduction

mutated at one of its four hotspot codons hyperactivates many downstream effector pathways . mutations reduce rate of intrinsic and/or GAP-mediated hydrolysis . 146 mutations do not alter rate of nucleotide exchange, but cause hyperactivation . a meta-analysis suggested that COAD tumors with a KRAS G13D allele were sensitive to anti-EGFR therapies . it is also possible that allele-specific genetic interactions drive the varying clinical outcomes . understanding the heterogeneous properties is essential to effectively treating kRAS-driven cancers .

## Results

### KRAS alleles are non-uniformly distributed across cancers

Across all the alleles, KRAS was most frequently mutated in PAAD (86%) most mutations by single-nucleotide substitutions occurred at one of four "hotspot" codons: 12, 13, 61, and 146. codon 12 mutations accounted for 72.2% of all mutations in the dataset . codon 13 (9.8%), 61 (14.8%), and 146 (3.2%) were the most frequent . there was substantial variability of the alleles found at these hotspots .

### The KRAS alleles have different mutagenic origins

all single-nucleotide mutations can be represented by the combination of six possible base substitutions . the most common mutational signatures were the “clock-like” single base substitution (SBS) signatsures SBS1 and SBS5 . mutational frequency of KRAS was not caused primarily by distinct compositions of mutational processes . each mutational process has a different propensity to induce each KRAS allele . the probability that the allele in an individual tumor was caused by any detectable process was based on the trinucleotide context . specific mutational signatures were more likely to have caused the observed mutation than expected . for example, in COAD and PAAD, SBS18 was strongly associated with G12C mutations . this corroborates previous finding that KRAS mutations are more frequent in patients with MUTYH .

### The frequency of most KRAS alleles cannot be solely attributed to the prevalence of detected mutagens

the null hypothesis tested was that the frequency of the KRAS alleles would be determined by the mutational processes alone . the average predicted frequencies across the samples of each cancer were compared against the observed allele frequencies . in COAD, G13D was predicted to be the most frequent allele (27%) but was observed less frequently (20%) in PAAD, all of the alleles were observed at a significantly different frequency than predicted by mutational signatures . the frequency of the most common allele, G12C, was substantially underestimated . the alleles never or rarely found in a cancer were predicted to occur at frequencies ranging from 1.5% (for Q61L in PAAD) to 10.5% . indicating that these allele are not rare because their causative mutations do not occur . instead, they are because of weak oncogenic fitness in the tissue . the probability of acquiring a particular KRAS allele was not significantly greater in tumor samples with a specific mutation . however, the same was not true for the mutations in COAD and LUAD . the cause of the mutation is currently unknown, though this result indicates that it plays an important role in mutagenesis. how the allele interacts with the preexisting signaling context of the tissue is likely a crucial factor in determining its frequency in cancer . how the unique biological properties of an allele interacted with the tissue warrants further investigation into their genetic interactions .

### The KRAS alleles have distinct comutation networks

increased frequency of comutation with another gene suggests cooperative effect . reduced frequency suggests second event is functionally redundant or inhibitory . mutational activation of EGFR demonstrated to be cytotoxic in the presence of a KRAS mutant . comutation interactions between each KRAS allele and every other mutated gene were investigated using a one-sided Fisher’s exact test of association . to reduce the number of false positive interactions, multiple filters were applied to restrict which genes were tested, including only testing for increased or reduced interactions . the result of the analysis on COAD tumors was a weakly connected network of the alleles with only a few genes linking them together . several KRAS alleles had reduced comutation with NRAS and BRAF . PIK3CA mutations tend to occur in several hotspots, each likely having slightly different effects on hyperactivation of the protein . some of the allele's showed enrichment for cellular functions in their networks . KRAS allele-specific comutation network uncovered in LUAD was far larger than that of COAD . this was likely caused by the higher mutation frequency in this cancer . there were several intriguing cellular processes enriched for each allele . some detectable comutation events were mutations acquired by distinct populations in a single patient . despite this caveat, there was no detectable pattern of comutations between particular KRAS and NRAS alleles . many genes had detectable comutation interactions with multiple alleles . of these, four interact with or signal through KRAS or are known oncogenes or tumor suppressors. TP53 was primarily mutated at known hotspots R175, R248, R273, and R282 . many of the comutation interactions identified from this allele-specific analysis were not identified from a gene-level analysis that disregards the KRAS allele information . Among these, only 35 were in common (Supplementary Fig. 8a)

### KRAS allele-specific genetic dependencies reveal potential synthetic lethal vulnerabilities

perturbations necessary to drive cancer expose vulnerabilities not present in normal cell-of-origin . data from a genome-wide, CRISPR/Cas9 knockout screen of cancer cell lines . analysis restricted to KRAS alleles for which there were at least three different cell lines with mutation . strong patterns in differential dependency of various cellular processes . genes involved in ERBB4 signaling tended to have a weaker lethal effect when knocked out . knocking out LIN7C had a more severe reduction on growth in KRAS G13D cell lines . G12D cell lines demonstrated a reduced dependency on the genes at the G2 and M DNA damage checkpoint . the G12R cell lines were less dependent on PI3K signaling downstream of FGFR1 . meanwhile, the cell lines with KRAS G12V mutations were less sensitive to the knockout of genes implicated in cellular senescence .

## Discussion

the cancer-specific distributions of KRAS mutations were influenced, but not determined, by the active mutational processes in the tumor samples . this suggests that the biological properties of the kRAS alleles, within the context of the tissue-of-origin, is an important factor in the positive selection of a mutation . analysis of KRAS genetic networks highlights tissue-specific nature of genetic interactions . allele-specific genetic interactions were not consistent between tissues, authors say . configuration of tissue signaling network influences interactions that arise during cancer progression . authors: somatic missense mutations that activate oncogenes are not always equivalent . the KRAS allele-specific comutation analysis indicates that the mutations act within distinct genetic environments . this likely impacts the effects of therapeutics, potentially obfuscating the underlying reason for disparate responses . the study has broad implications for the understanding of oncogene biology and for cancer therapy .
